### PR TITLE
Add badges to README

### PR DESCRIPTION
**Pull request by willingc**
- Add snyk badge since it reflects well on the project
- ~Add python 3.9 badge~
- Add python 3.9 to setup metadata

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@
 [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/)
 [![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/downloads/release/python-370/)
 [![Python 3.8](https://img.shields.io/badge/python-3.8-blue.svg)](https://www.python.org/downloads/release/python-380/)
+[![Python 3.9](https://img.shields.io/badge/python-3.9-blue.svg)](https://www.python.org/downloads/release/python-390/)
```
Comment (Borda) [inline README.md:14]: [PyPI - Python Version](https://pypi.org/project/papermill/)
```
[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/papermill)](https://pypi.org/project/papermill/)
```

**Review by Borda — COMMENTED** [review]
rather take the details dynalically from pypi meta information

File: README.md (hunk 1 above)
Comment (MSeal) [reply README.md:14]: That's a nice change so it picks up when we update the build tags. Thoughts @willingc ?

Comment (willingc) [reply README.md:14]: Love it @Borda. I wasn't aware of that one option

**Comment by willingc** [comment]
@MSeal I added an explicit codecov config file to `.github` directory to resolve upload to codecov errors.

**Pull request merged**